VERSIONING.md

```diff
@@ -0,0 +1,93 @@
+# Versioning
+
+## SemVer and shorter SemVer
+
+We use [semantic versioning] for all of our images. We also support a shorter
+version of semantic versioning. E.g. `v<major> and v<major>.<minor>`.
```

Comment:
worth mentioning that fully specified X.Y.Z image is immutable and will never be changed.